database/buildSchema/34_row_level_policy_related.sql

```diff
@@ -3,8 +3,8 @@ ALTER TABLE public.application ENABLE ROW LEVEL SECURITY;
 DO $$
 BEGIN
   CREATE ROLE graphile_user WITH NOLOGIN;
```

Comment:
now that you have drop user, we shouldn't need this condition/exception

Reply:
Okay, does that mean I can replace the whole function block:
```
DO $$
BEGIN
  CREATE ROLE graphile_user WITH NOLOGIN;
  EXCEPTION WHEN DUPLICATE_OBJECT
  THEN RAISE NOTICE 'not creating role graphile_user -- it already exists';
END
$$;
```
with just:
```
CREATE ROLE graphile_user WITH NOLOGIN;
```
??

Reply:
yep, it worked for me when i tested it